external/common/requirements.txt

```diff
@@ -15,7 +17,8 @@ unittest2==0.5.1
 validictory==0.9.1
 PyMySQL==0.6.2
 DBUtils==1.1
-numpy>=1.7.1
+numpy>=1.9.2
 tweepy==2.1
 pyproj==1.9.3
 prettytable==0.7.2
+nupiccore-python==1.0
```

Comment:
Since this is the Python package name, the "-python" seems redundant. And we should probably start is as version 0.1.0 until we consider it stable.